README.md

```diff
@@ -113,12 +113,25 @@ or in development:
 * `flotiqStarterUrl` - full link to GatsbyJs starter, the list below
 * `wordpressUrl` - full link to WordPress site from which you wish to migrate content to Flotiq
 * `options` - additional options for command:
-    * `withInternal=1` - purge should remove also internal type objects (`_media`)
+* `withInternal=1` - purge should remove also internal type objects (`_media`)
 
 ### Flags
 
 `--json-output`, `-j` - Error and console output will be additionally written into json file named `output.json`.
 
+### Install Flotiq SDK
+
+Execute:
+`flotiq sdk install [directory] [language] [flotiqApiKey]`
```

Comment:
Here is
`[directory] [language]` but in this file https://github.com/flotiq/flotiq-cli/pull/44/files#diff-1f16739920c8b9eff1b1e6b3c6d94d55b7fcdf6809599bf6ddbd96709a3b8c46R137 is
`[language] [directory]` (different order)